### ISSUE TITLE
Location restrictions in v3 manifest only for images

### ISSUE THREAD

**Issue by LynnMcRae** [sul-dlss/purl#180]
Part of the UV support work was to extend location restrictions to all content types. It appears however that we only generate location restriction information in the v3 manifest for resources that are themselves images, ignoring media/doc/etc resources also (or exclusively) in the canvas.  The only reference to `iiif_location_auth_service`  is here:
https://github.com/sul-dlss/purl/blob/691de6776a4cda9815cc1eb47f61bbad3b30185b/app/models/iiif3_presentation_manifest.rb#L102-L122

Compare https://sul-purl-uat.stanford.edu/bb099mt5053/iiif3/manifest
where the location restriction appears with the accessory image, but not on the media itself which is also restricted...

with https://sul-purl-uat.stanford.edu/yv090xk3108/iiif3/manifest 
where the restriction is not expressed at all because the only resource is the PDF.

**Comment by cbeer** [comment]
I don't believe they've tackled iiif auth for non-image resources yet (not to mention that authorization information in manifests is non-authoritative; clients MUST validate it against the `info.json`, which our non-image resources don't have.)

**Comment by LynnMcRae** [comment]
I think all we're looking for is information about the restriction as reflected in the manifest in order to generate a heads up for the user in Embed, not the auth information in info.json, but I could be wrong about where UV is looking for that information. Will follow up with Digitrati and Stu.
